starter_code/components.js

```diff
@@ -0,0 +1,41 @@
+class Picture extends React.component {
+  constructor(){
+    super();
+    this.state = { image: 'http://images.fanpop.com/images/image_uploads/wooderson---Slater-dazed-and-confused-301811_400_300.jpg'}
```

Comment:
Would it have been helpful to make this `image` a boolean? Then we could toggle between true/false instead of emptying out the image string to blank when the state changes. That would probably clean up your code a bit too.